httpserver/http_connection.cpp

```diff
@@ -538,15 +580,19 @@ void connection_t::process_swarm_req(std::string_view target) {
         }
 
         const auto it = header_.find(OXEN_SENDER_SNODE_PUBKEY_HEADER);
-        if (it != header_.end()) {
-            const std::string& tester_pk = it->second;
-            this->process_storage_test_req(blk_height, tester_pk, msg_hash);
-        } else {
+        if (it == header_.end()) {
             OXEN_LOG(debug, "Ignoring test request, no pubkey present");
+            return;
+        }
+        auto tester_pk = parse_pubkey(it->second);
+        if (!tester_pk) {
+            OXEN_LOG(debug, "Ignoring test request, invalid pubkey");
```

Comment:
we still need to respond with http 400 here.

Reply:
Arghhh, why is this even being send over HTTP?

Reply:
It actually look like it's the above ones that are misleading because at the top of all of this the response has already been set to bad_request so they aren't actually doing anything.  I've removed the redundant ones.